go.mod

```diff
@@ -10,6 +10,8 @@ require (
 	github.com/prometheus/client_golang v1.5.1
 	github.com/stretchr/testify v1.5.1
 	go.uber.org/zap v1.14.1
+	golang.org/x/net v0.0.0-20200301022130-244492dfa37a // indirect
```

Comment:
this got auto generated from `go mod tidy`?

Do not see any new dependencies

Reply:
yes derives from `go mod tidy`